src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques6 {
+    private String model;
+    private List<?> messages;
```

Comment:
Using wildcard types like List<?> can lead to type safety issues. Consider specifying the type explicitly.